schemas/collector-report.md

```diff
@@ -0,0 +1,67 @@
+# The collector report JSON schema
+
+This document details the structure of reports collected by https://github.com/foolip/mdn-bcd-collector and stored in https://github.com/foolip/mdn-bcd-results.
+
+## JSON structure
+
+Below is an example of the report data:
+
+```json
+{
+  "__version": "6.2.7",
```

Comment:
To add the `extensions` property, and also to reference the latest version of the collector:

```suggestion
  "__version": "9.3.0",
  "extensions": {},
```